ocp_resources/user_defined_network.py

```diff
@@ -66,3 +68,78 @@ def to_dict(self) -> None:
                 _spec["localNet"] = self.local_net
 
     # End of generated code
+
+
+class TopologyType:
```

Comment:
Any reason why this is a class and not constant?

Reply:
I chose to implement it as a class because I plan to add additional topology types (like LAYER3) in future updates. Using a class allows for a structured and scalable approach to managing these constants.

Reply:
I added the l3 constant.

Reply:
`LAYER3` is not used in the code, remove it.

When to resolve/not resolve comments:
1. If you changed the code as the commenter asked you should resolve the comment.
2. if you do not agree, write your comment and do not resolve the comment.
3. if you change the code but not as the commenter asked, do not resolve the comment.

Reply:
Got it, I removed the LAYER3 and refactor to use regular constant instead of a class.
I will refactor it again to use class in the layer3 PR.